src/main/java/spireMapOverhaul/zones/hailstorm/events/AbandonedCamp.java

```diff
@@ -0,0 +1,174 @@
+package spireMapOverhaul.zones.hailstorm.events;
+
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.actions.common.HealAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.AbstractEvent;
+import com.megacrit.cardcrawl.events.AbstractImageEvent;
+import com.megacrit.cardcrawl.helpers.*;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import com.megacrit.cardcrawl.relics.DeadBranch;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.hailstorm.cards.Freeze;
+import spireMapOverhaul.zones.hailstorm.relics.Flint;
+
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class AbandonedCamp extends AbstractImageEvent {
+    public static final String ID = SpireAnniversary6Mod.makeID(AbandonedCamp.class.getSimpleName());
+    private static final EventStrings eventStrings = CardCrawlGame.languagePack.getEventString(ID);
+    private static final String NAME = eventStrings.NAME;
+    private static final String[] DESCRIPTIONS = eventStrings.DESCRIPTIONS;
+    private static final String[] OPTIONS = eventStrings.OPTIONS;
+
+    public static final String IMG = SpireAnniversary6Mod.makeImagePath("events/Hailstorm/AbandonedCamp.png");
+
+    private static final float FLINT_HP_LOSS = (float) 1 /24;
+    private static final float A_15_FLINT_HP_LOSS = (float) 1 /16;
+    private static final int GOLD = 90;
+    private static final int A_15_GOLD = 60;
+    private static final float HEALING = (float) 1/6;
+    private static final float A_15_HEALING = (float) 1/9;
+
+    private final boolean T_DEADBRANCH_F_FLINT;
+    private final boolean T_GOLD_F_HEALING;
+
+    private final AbstractCard curse = new Freeze();
+    private int hpLoss;
+    private int goldGain;
+    private int healing;
+
+    private int screenNum = 0;
+
+
+    public AbandonedCamp() {
```

Comment:
I think it'd be better if you made a custom relic instead of dead branch. It can be a simple one like `The first time you deal damage each combat, deal 12 damage to ALL enemies.` for like a frozen stick.

Just think that Dead Branch is very polarizing and giving direct access to it is something wed'd like to avoid.

Reply:
We alr went through this discussion. I personally wanted to give access to this relic as it is very thematic on top of being fun.
It is not a guarantee drop.
Also I think people created way more polarizing stuff (mainly from my experience with PM i didn't play much biomes).

I think it will bring more enjoyment than cutting it